apps/video/js/video.js

```diff
@@ -827,13 +827,10 @@ function movePlayHead(percent) {
 }
 
 function updateVideoControlSlider() {
-  // We update the slider when we get a 'seeked' event.
-  // Don't do updates while we're seeking because the position we fastSeek()
-  // to probably isn't exactly where we requested and we don't want jerky
-  // updates
-  if (dom.player.seeking) {
-    return;
-  }
```

Comment:
I wonder if the real issue here was that by the time we got one seeked event, we had already called fastSeek() again, so we weren't updating the slider even when we did get the seeked events.